internal/rpc/jsonrpc/methods.go

```diff
@@ -541,6 +542,36 @@ func (s *Server) dumpPrivKey(ctx context.Context, icmd interface{}) (interface{}
 	return key, nil
 }
 
+// fundRawTransaction adds inputs  matching the total output value to a transaction.
+func (s *Server) fundRawTransaction(ctx context.Context, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*types.FundRawTransactionCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, errUnloadedWallet
+	}
+
+	txBytes, err := hex.DecodeString(cmd.HexString)
+	if err != nil {
+		return nil, err
+	}
+
+	var tx wire.MsgTx
+	if err := tx.FromBytes(txBytes); err != nil {
+		if errors.Is(err, errors.Locked) {
```

Comment:
The error being handled is a tx deserialization error. Why would you check for a wallet-is-locked error here?